idea_town/frontend/static-src/app/views/header-view.js

```diff
@@ -30,20 +29,15 @@ export default BaseView.extend({
   //       to pick up csrftoken cookie changes.
   logout(evt) {
     evt.preventDefault();
-    xhr({
+    fetch(app.getDocHeadLink('x-logout', 'href'), {
```

Comment:
Piggybacking on my last comment there, we've also got the addon URL specified via `app.me.addon.url`.

Would be nice to unify all that at some point.